src/main/java/bio/terra/testrunner/runner/config/TestConfiguration.java

```diff
@@ -59,7 +59,7 @@ public static TestConfiguration fromJSONFile(String resourceFileName) throws Exc
       testConfig.testUsers.add(testUser);
     }
 
-    // instantiate default kubernetes, application specification objects, if null
```

Comment:
I don't follow the comment update -- we are still instantiating the application specification object below if it's null.

Reply:
Restored the original comment.